src/bin/console/console.cpp

```diff
@@ -211,14 +212,13 @@ int Console::_eval_command(const CommandFunction& func, const std::string& comma
 }
 
 int Console::_eval_sql(const std::string& sql) {
-  SQLQueryPlan plan;
-  hsql::SQLParserResult parse_result;
+  _sql_pipeline = std::make_unique<SQLPipeline>(sql, _explicitly_created_transaction_context);
```

Comment:
Since the Pipeline now handles the TransactionContext, shouldn't we just let this be completely handled by the Pipeline, so that we don't need to manage one in the Console in addition?

Reply:
You might have multiple pipelines per transaction (see my comment above)